ibm/mas_devops/common_vars/compatibility_matrix.yml

```diff
@@ -117,5 +117,4 @@ upgrade_path:
 
 aiservice_upgrade_path:
   9.2.x: 9.2.x
```

Comment:
why is there a `9.2.x: 9.2.x` ? I know you didn't add it but i am not sure why that is there